compiler/rustc_middle/src/mir/interpret/error.rs

```diff
@@ -439,6 +439,8 @@ pub enum InterpError<'tcx> {
     /// The program exhausted the interpreter's resources (stack/heap too big,
     /// execution takes too long, ...).
     ResourceExhaustion(ResourceExhaustionInfo),
+    /// The program terminated immediately from a panic.
+    Panic(Symbol),
```

Comment:
This is a CTFE-only error, I don't think it should be added here. This is more of a case for the `MachineStopType`.